frontend/packages/dev-console/src/utils/imagestream-utils.ts

```diff
@@ -87,7 +87,7 @@ export const normalizeBuilderImages = (
 ): NormalizedBuilderImages => {
   const builderImageStreams = imageStreams.filter((imageStream) => isBuilder(imageStream));
 
-  return builderImageStreams.reduce((builderImages, imageStream) => {
+  return builderImageStreams.reduce((builderImages: NormalizedBuilderImages, imageStream) => {
```

Comment:
```

ERROR in /go/src/github.com/openshift/console/frontend/packages/dev-console/src/utils/imagestream-utils.ts
(90,3): Type 'K8sResourceKind' is not assignable to type 'NormalizedBuilderImages'.
  Property 'apiVersion' is incompatible with index signature.
    Type 'string' is not assignable to type 'BuilderImage'.
```

this type error appeared after the changes of `apiVersion` -> `apiVersion?`. Not sure why this triggered it but the type has to be specified for the `accumulator` in `reduce` function.